src/settings-ui/Settings.UI.Library/ViewModels/PowerLauncherPluginViewModel.cs

```diff
@@ -189,5 +200,27 @@ public bool ShowNotAllowedKeywordWarning
         {
             get => !Disabled && NotAllowedKeywords.Contains(ActionKeyword);
         }
+
+        private static readonly List<string> MathematicKeywords = new List<string>()
+        {
+            // Using one of them breaks global queries for calculator plugin.
+            // ".", ",", "+", "-", "(", => We do not warn on dot as we use it currently for the program plugin and there are no issues regarding the dot opened by our users. (htcfreek, 2022-07-24)
+            ",", "+", "-", "(",
```

Comment:
numbers can be set as plugin activation command. Include numbers here as well?

Reply:
Looks like we need a more complex regex here. Do you agree?

Reply:
Yes. Let's include those as well..

Reply:
Then I suggest to have a "fixed" list of strings and the following regex condition:
- `[a-zA-Z]*\(?\d*[\.\,]?\d*\)?` (pi(0); pi(5.0); pi(6,7); pi)